docs/en/api/FS-API.md

```diff
@@ -353,120 +359,13 @@ Alluxio has a [Python Client](https://github.com/Alluxio/alluxio-py) for interac
 [REST API](#rest-api). The Python client exposes an API similar to the [Alluxio Java API](#java-client).
 See the [doc](http://alluxio-py.readthedocs.io) for detailed documentation about all available
 methods. See the [example](https://github.com/Alluxio/alluxio-py/blob/master/example.py) on how to perform basic
-filesystem operations in Alluxio.
-
-### Alluxio Proxy dependency
-
-The Python client interacts with Alluxio through the REST API provided by the Alluxio proxy.
-
-The proxy is a standalone server that can be started using
-`${ALLUXIO_HOME}/bin/alluxio-start.sh proxy` and stopped using `${ALLUXIO_HOME}/bin/alluxio-stop.sh
-proxy`. By default, the REST API is available on port 39999.
-
-There are performance implications of using the HTTP proxy. In particular, using the proxy requires
-an extra hop. For optimal performance, it is recommended to run the proxy server and an Alluxio
-worker on each compute node.
```

Comment:
Duplicate content.

Reply:
okay with removing duplicate content but it should still be contained somewhere in this doc for the python and go client sections to point to. i agree it is not good to have two copies but it is also not good to have zero copies

Reply:
Yes there is one copy left. See here: https://github.com/Alluxio/alluxio/blob/90fc2207eaee1813c4f66ccacc3bdc6308046b8d/docs/en/api/FS-API.md#L345-L351

These are the duplicates:
https://github.com/Alluxio/alluxio/blob/90fc2207eaee1813c4f66ccacc3bdc6308046b8d/docs/en/api/FS-API.md#L365-L371
https://github.com/Alluxio/alluxio/blob/90fc2207eaee1813c4f66ccacc3bdc6308046b8d/docs/en/api/FS-API.md#L485-L491

I will add a pointer to that section under the python and go sections, so that impatient people have a chance to read about the performance implications.

Reply:
great!